### ISSUE TITLE
Different comment form for admin users

### ISSUE THREAD

**Issue by katjahenttonen** [City-of-Helsinki/kerrokantasi-ui#675]
When a KerroKantasi admin is leaving a comment, the comment form should look slightly different from the form of an ordinatory user.  Anybody who is entitled to open new hearing at KerroKantasi is herein regarded as an admin.  The attached picture shows how the admin  comment form is supposed to look like (please ignore some wrong fonts in the picture, all fonts are supposed to look the same). Instead of  the subtitle "nimimerkki" , there is a subtitle "nimi ja organisaatio" (ignore the wrong font in the later part of the title). There  is also a yellow info message telling "Olet kirjautuneena pääkäyttäjänä.  Nimesi ja organisaatiosi näytetään kommenttisi yhteydessä".  The name field and organization are  automatically filled and should not  NOT be changed by the user (ie. fields must be locked in the UI).  However, there is a checkbox entitled "Piilota nimi"  and ticking that box shoud replace the emplyee's real name with the word "Työntekijä" in the name field. Unticking the box should put back the users's real name into the name field.

![pääkäyttäjänkommentti](https://user-images.githubusercontent.com/39044068/58416067-16a07480-8089-11e9-815a-d60352fd7b59.png)

Admins can also answer other comments as shown in #674 and the form will be the same as presented above (except of opening up in a different location, ie. in the middle of the comment thread).

**Comment by farrukh8925** [comment]
A couple of questions here again:
What property in the user response determines a city employee. For instance, in my case there is this param "provider" and valued as helsinki, is that the param ?

Second, where do we get the list of available organizations ?

@katjahenttonen

**Comment by farrukh8925** [comment]
<img width="448" alt="Screenshot 2019-05-27 at 12 31 30" src="https://user-images.githubusercontent.com/49946648/58410730-6a0bc600-807b-11e9-8817-b6b8e089b113.png">

**Comment by katjahenttonen** [comment]
@farrukh8925  the description of the issue has now been changed according to what we discussed with Riku on Slack on friday.  The comment form will be different only for KerroKantasi admins and not for all Helsinki city  employees.  The name field and organizations field are both filed automatically and therefore we do not need the list of available organizations.

**Comment by Rikuoja** [comment]
Yeah, `adminOrganizations` is the field.

**Comment by Rikuoja** [comment]
However, this issue has the problem that currently the API has no data on the comment author organization. It will have to be saved in every comment so the UI can display it.

I will make an API issue on it when I have the time.

**Comment by Rikuoja** [comment]
I would suggest that to the comment JSON 
```
"section": "ZMSHnfKMkPG1vGfCwxOWIq7CgMCk1cfM",
            "language_code": "fi",
            "answers": [],
            "id": 27185,
            "content": "A vaihtoehto paras, koska edes yksi tehdasrakennuksen säästetty.\nSuunnitelmat talot liian korkeita. Kuka haluaa/voi asua junanradan vieressä? Tärinää ja melua, jotka tutkimuksissa havaittu asumiselle sopimattomaksi. Ankeaa.",
            "author_name": "Anonyymi",
            "n_votes": 0,
            "created_at": "2019-05-27T08:03:50.008292Z",
            "is_registered": false,
            "can_edit": false,
            "geojson": null,
            "images": [],
            "label": null,
            "hearing": "jzMRBXnb58fZ4XelGASkXMFXGw4wfkZJ"
```

we would add a new field `"organization"` that contains the organization name.

However, currently the API has no API endpoint and no organization ids available to the users. Best would be if the API could link the comment to the specific organization as well as the user. This would mean that we add an organization endpoint with organization ids, and then refer to organization ids here (not their names).

**Comment by katjahenttonen** [comment]
@farrukh8925  as discussed on Slack previously, the comment form should open in the middle of the thread when the user clicks "Vastaa".  However, the newly added comment should be placed at the end of thread and the screen should be scrolled down so that the user sees the comment she/he added.

**Comment by Rikuoja** [comment]
Actually, now the field `organization` exists in the backend comment. The UI cannot edit the comment `organization` field anyway, because it is filled by the backend if the user is admin.

Therefore, we can just show a different form depending on user `adminOrganizations` here, and display the first admin org on the form. No need to POST the `organization` field in a comment, so this is ready to be coded.

**Comment by Rikuoja** [comment]
Solved in #685
